### ISSUE TITLE
Error when running commands in Python 3

### ISSUE THREAD

**Issue by dafnevk** [nlppln/nlppln#27]
When I try running any command in Python 3.6, I get the following error:

```bash
python -m nlppln.commands.ls .
Traceback (most recent call last):
  File "/home/dafne/anaconda2/envs/adh/lib/python3.6/runpy.py", line 183, in _run_module_as_main
    mod_name, mod_spec, code = _get_module_details(mod_name, _Error)
  File "/home/dafne/anaconda2/envs/adh/lib/python3.6/runpy.py", line 153, in _get_module_details
    code = loader.get_code(mod_name)
AttributeError: 'Py2Fixer' object has no attribute 'get_code'
```

**Comment by dafnevk** [comment]
More details of the problem (no solution yet):
- When running a module with `python -m <module>` the script `runpy.py` tries to access `get_code` from the module loader. 
- The loader is Py2Fixer and this loader does not have the `get_code` function

**Comment by dafnevk** [comment]
Correction, the problem lies in the package `salad`:

```
~/anaconda2/envs/nlppln3/lib/python3.6/site-packages/cwltool/load_tool.py in <module>()
     17 from six.moves import urllib
     18 
---> 19 import schema_salad.schema as schema
     20 from avro.schema import Names
     21 from ruamel.yaml.comments import CommentedMap, CommentedSeq

~/anaconda2/envs/nlppln3/lib/python3.6/site-packages/schema_salad/__init__.py in <module>()
     29                 exit(1)
     30 
---> 31     from past import autotranslate  # type: ignore
     32     autotranslate(['avro', 'avro.schema'])

ModuleNotFoundError: No module named 'past'
```

the `autotranslate` function 'hooks' the Py2Fixer loader into sys.meta_path.  That only happens after scriptcwl is imported, and through it cwltool and salad.

The reason it only happens for the modules in `commands` is that `runpy.py` loads the parent module, and in `nlppln/__init__.py` the dependencies are loaded.

**Comment by dafnevk** [comment]
I created a pull request in schema_salad:
https://github.com/common-workflow-language/schema_salad/pull/165

**Comment by dafnevk** [comment]
Fixed in schema_salad version 2.7.20180322174040.
